src/Admin/FieldDescriptionInterface.php

```diff
@@ -50,7 +50,7 @@ public function getName();
      * @param string     $name
      * @param mixed|null $default
      *
-     * @return array|null the value represented by the provided name
+     * @return mixed the value represented by the provided name
```

Comment:
According to PHPDoc, does `mixed` include `null` values?

Reply:
Yes, see https://docs.phpdoc.org/guides/types.html

Reply:
```
mixed
    A value with this type can be literally anything; the author of the documentation is unable to predict which type it will be.
```

IMHO *anything* doesn't mean nothing (null).

Reply:
I think nothing is better representing by void, also, if you look at the list of types, null is in it, while void isn't.

> The value contained, or returned, is literally null. This type is not to be confused with void, which is the total absence of a variable or value (usually used with the @return tag).